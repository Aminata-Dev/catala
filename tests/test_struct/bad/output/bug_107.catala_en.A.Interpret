[ERROR] struct name "S" already defined

First definition:
 --> tests/test_struct/bad/bug_107.catala_en
  | 
4 | declaration structure S:
  |                       ^
  + https://github.com/CatalaLang/catala/issues/107

Second definition:
 --> tests/test_struct/bad/bug_107.catala_en
  | 
8 | declaration structure S:
  |                       ^
  + https://github.com/CatalaLang/catala/issues/107
