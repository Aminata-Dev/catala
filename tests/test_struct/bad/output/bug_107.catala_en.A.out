[ERROR] struct name "S" already defined

First definition:
 --> test_struct/bad/bug_107.catala_en
  | 
4 | declaration structure S:
  |                       ^
  + https://github.com/CatalaLang/catala/issues/107

Second definition:
 --> test_struct/bad/bug_107.catala_en
  | 
8 | declaration structure S:
  |                       ^
  + https://github.com/CatalaLang/catala/issues/107
